docs/api/cloudmarker.stores.rst

```diff
@@ -9,6 +9,14 @@ cloudmarker.stores package
 Submodules
 ----------
 
+cloudmarker.stores.emailalert module
```

Comment:
I think this needs to be corrected.

Reply:
Yes, there is issue #72 open to track this.